tools/testbench/common_test.c

```diff
@@ -119,7 +119,7 @@ int tb_pipeline_start(struct ipc *ipc, struct pipeline *p,
 		printf("Warning: Failed prepare pipeline command.\n");
 
 	/* Start the pipeline */
-	ret = pipeline_trigger(p, cd, COMP_TRIGGER_START);
+	ret = pipeline_trigger(p, cd, COMP_TRIGGER_PRE_START);
```

Comment:
Does the commnt need updated here too since we are not fully starting the pipeline ?